lab_5_scrapper/scrapper.py

```diff
@@ -13,62 +52,93 @@ def __init__(self, path_to_config: Path) -> None:
         """
         Initializes an instance of the Config class
         """
-        pass
+        self.path_to_config = path_to_config
+        self.config_dto = self._extract_config_content()
+        self._validate_config_content()
+        self._seed_urls = self.get_seed_urls()
+        self._headers = self.get_headers()
+        self._num_articles = self.get_num_articles()
+        self._get_encoding = self.get_encoding()
+        self._timeout = self.get_timeout()
+        self._should_verify_certificate = self.get_verify_certificate()
+        self._headless_mode = self.get_headless_mode()
 
     def _extract_config_content(self) -> ConfigDTO:
         """
         Returns config values
         """
-        pass
+        with open(self.path_to_config, 'r', encoding='utf-8') as infile:
+            reader = json.load(infile)
+        return ConfigDTO(**reader)
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters
         are not corrupt
         """
-        pass
+        if not self.config_dto.seed_urls or not isinstance(self.config_dto.seed_urls, list) \
+                or not all(isinstance(url, str) for url in self.config_dto.seed_urls):
+            raise IncorrectSeedURLError
+
+        if not isinstance(self.config_dto.total_articles_to_find_and_parse, int) \
+                or not 1 < self.config_dto.total_articles_to_find_and_parse < 150:
+            raise IncorrectNumberOfArticlesError
+
+        if not self.config_dto.seed_urls and not isinstance(self.config_dto.headers, dict) \
+                and not all(isinstance(key, str) and isinstance(value, str)
+                            for key, value in self.config_dto.headers.items()):
+            raise IncorrectHeadersError
+
+        if not isinstance(self.config_dto.encoding, str):
+            raise IncorrectEncodingError
+
+        if not isinstance(self.config_dto.timeout, int) or self.config_dto.timeout < 1 or self.config_dto.timeout > 60:
```

Comment:
consider using constants from `core_utils.constants` for the upper and lower limits